library/parse_backup_metadata.py

```diff
@@ -48,6 +49,10 @@ def parse_backup_metadata(params):
     puppetserver_present = bool(find_rpm(rpms, "^puppetserver-[\d+].*"))
     qpidd_present = bool(find_rpm(rpms, "^qpid-cpp-server-[\d+].*"))
 
+    if os.getenv('SATELLITE_CLONE_ALLOW_KATELLO'):
```

Comment:
Should it automatically fall back to this if `satellite_version` is empty (as it would be in upstream)?
```suggestion
    if not satellite_version:
```

Reply:
Right now, this is `satellite-clone`, and it has safeguards to verify it actually is a Satellite it's cloning.
Katello is not Satellite, and I'd prefer to keep this for now (and probably never support it in this form, but teach foreman-maintain how to clone both kinds of setups).

The env var makes it explicitly "user opt-in"